src/meander/match/epsilon.cljc

```diff
@@ -530,24 +531,19 @@
        (compile-pass targets [row])
 
        :jsa
-       (if (literal? node)
```

Comment:
Literal conversion needs to occur in `meander.match.syntax.epsilon/expand-ast-top-down` with respect to a flag.

Reply:
Actually, this was a mistake. Reverting.